packages/common/src/components/DeactivatedAccount.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import styled from "styled-components";
+import IconButton from "./IconButton";
+import { IconArrowRight } from "hds-react";
+import { fontBold, fontRegular } from "../common/typography";
+
+const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: var(--container-width-xl);
+  margin: 0 auto;
+  gap: 42px;
+  padding-block: 42px;
+  padding-inline: var(--spacing-s);
+  --spacing-hz: var(--spacing-s);
+  img {
+    width: 185px;
+    height: auto;
+    margin: 0 auto;
+  }
+  @media (min-width: 768px) {
+    flex-direction: row;
+    padding-inline: var(--spacing-m);
+    --spacing-hz: var(--spacing-m);
+    img {
+      display: flex;
+      margin: 0;
+      margin-inline: auto;
+      width: auto;
+      max-height: 418px;
+      max-width: 350px;
+      flex-grow: 1;
+    }
+  }
+`;
+
+const TextContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 42px;
+  justify-content: center;
+  @media (min-width: 768px) {
+    order: -1;
+  }
+`;
+
+const Heading = styled.h1`
+  margin: 0;
+  line-height: 1;
+  font-size: 40px;
+  ${fontRegular}
```

Comment:
nope. we don't modify the fonts per page for `h1`

inherit from `H1` and it's what it is on other pages.